Makefile

```diff
@@ -19,7 +19,7 @@ docker-build:
 	docker build -t ursa -f ./Dockerfile .
 
 docker-run:
-	docker run -p 4069:4069 -p 4070:4070 -p 6009:6009 -p 8070:8070 --name ursa-cli -it ursa
+	docker run --name ursa-cli -it ursa
```

Comment:
still need 6009 right?

Reply:
users who want that exposed should use the full-node impl